btr-web/btr-main-app/utils/btr-bods/si-schema-to-btr-bods-converters.ts

```diff
@@ -331,15 +331,15 @@ const _getInterestsOfSharesOrVotes = (
   return interests
 }
 
-const getInterests = (si: SiSchemaType) => {
+const getInterests = (si: SiSchemaType, force: boolean = false) => {
```

Comment:
Instead of 'force' could we call this 'includeUnchanged' or 'includeAll' to make it a little more clear what its for?